solana/solana-ibc/programs/solana-ibc/src/lib.rs

```diff
@@ -637,6 +666,21 @@ pub struct SendTransfer<'info> {
     system_program: Program<'info, System>,
 }
 
+#[derive(Accounts)]
+#[instruction(new_length: usize)]
+pub struct ReallocAccounts<'info> {
+    #[account(mut)]
+    payer: Signer<'info>,
+
+    #[account(mut)]
+    /// CHECK:
+    account: UncheckedAccount<'info>,
+
+    rent: Sysvar<'info, Rent>,
```

Comment:
This isn’t needed since we’re accessing rent via `Rent::get()`.

Reply:
right